### PR TITLE
1.Addition

### PR DESCRIPTION
**Pull request by ameyunaikgithub**

### CODE REVIEW

File: addition/addition.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const addition =(a,b) => {
```
Comment (allopez7) [inline addition/addition.js]: Add more descriptive names for a, b.

File: addition/addition.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  c = a+b;
```
Comment (allopez7) [inline addition/addition.js]: just return a+b

File: addition/addition.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  return c;
```
Comment (allopez7) [inline addition/addition.js]: get rid of it

**Review by allopez7 — CHANGES_REQUESTED** [review]
Check comments!

**Review by kinuesantos — APPROVED** [review]

**Review by allopez7 — APPROVED** [review]
Good Job!

**Pull request merged**